src/main/java/fr/esgi/al5_2/Tayarim/auth/JwtHelper.java

```diff
@@ -18,75 +18,96 @@
 @Component
 public class JwtHelper {
 
-    @Value("${PRIVATE_KEY}")
-    private String privateKey;
-
-    public String generateToken(String email, String uuid) {
-        String subject = email.concat(";").concat(uuid);
-        var now = Instant.now();
-        return Jwts.builder()
-                .subject(subject)
-                .issuedAt(Date.from(now))
-                .expiration(Date.from(now.plusSeconds(60*60))) //60s * 60m = 1h
-                .signWith(getSignKey())
-                .compact();
-    }
+  @Value("${PRIVATE_KEY}")
+  private String privateKey;
 
-    public String extractSubject(String token) {
-        try{
-            Claims tokenBody = getTokenBody(token);
-            if(tokenBody == null){
-                return null;
-            }
-
-            return tokenBody.getSubject();
-        } catch (Exception e){
-            throw new TokenExpireOrInvalidException();
-        }
-    }
+  /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Summary javadoc is missing.